packages/admin-ui/src/lib/core/src/shared/components/facet-value-chip/facet-value-chip.component.ts

```diff
@@ -13,4 +13,9 @@ export class FacetValueChipComponent {
     @Input() removable = true;
     @Input() displayFacetName = true;
     @Output() remove = new EventEmitter<void>();
+
+    get formattedTitle(): string {
```

Comment:
One note: using getters in template bindings can result in _lots_ of calls. Try adding a console log in this getter and see. If it is resulting in excessive calls, consider saving the title to a regular class field and setting in in the `OnInit` lifecycle hook.

Reply:
https://github.com/vendure-ecommerce/vendure/pull/3175/commits/b499e03717164d0a0556f5edcb97aaebf0a80f4f